sdk/appcenter/src/main/java/com/microsoft/appcenter/AppCenter.java

```diff
@@ -655,6 +672,12 @@ private void setInstanceEnabled(boolean enabled) {
             StorageHelper.PreferencesStorage.putBoolean(PrefStorageConstants.KEY_ENABLED, true);
         }
 
+        /* Send started services. */
+        if (mStartedServicesNamesToLog != null && switchToEnabled) {
```

Comment:
Looks like 1 unit test is missing for code coverage

Reply:
Not for this line, I think